cmd/juju/common/controller.go

```diff
@@ -84,13 +86,23 @@ func WaitForAgentInitialisation(ctx *cmd.Context, c *modelcmd.ModelCommandBase,
 	}
 
 	ctx.Infof("Contacting Juju controller%s to verify accessibility...", addressInfo)
-	apiAttempts = 1
+	apiAttempts := 1
 	for attempt := attempts.Start(); attempt.Next(); apiAttempts++ {
 		err = tryAPI(c)
 		if err == nil {
-			ctx.Infof("Bootstrap complete, %q controller now available", controllerName)
-			ctx.Infof("Controller machines are in the %q model", bootstrap.ControllerModelName)
-			ctx.Infof("Initial model %q added", hostedModelName)
+			msg := fmt.Sprintf("\nBootstrap complete, controller %q now is available", controllerName)
+			if isCAAS {
+				msg += fmt.Sprintf(" in namespace %q", controllerName)
+				msg += `
+Now you can run 
+	juju add-model <model-name>
+to create a new model to deploy CAAS workload
```

Comment:
to create a new model to deploy k8s workloads

Reply:
fixed